docs/docs/integrations/retrievers/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 0
+sidebar_class_name: hidden
+---
+
+# Retrievers
+
+A **retriever** is an interface that returns documents given an unstructured query.
+It is more general than a vector store.
+A retriever does not need to be able to store documents, only to return (or retrieve) them.
+Retrievers can be created from vector stores, but are also broad enough to include [Wikipedia search](/docs/integrations/retrievers/wikipedia/) and [Amazon Kendra](/docs/integrations/retrievers/amazon_kendra_retriever/).
+
+Retrievers accept a string query as input and return a list of Document's as output.
+
+For specifics on how to use retrievers, see the [relevant how-to guides here](/docs/how_to/#retrievers).
+
+This table lists common retrievers.
+
+
+| Namespace 🔻 | Class |
```

Comment:
could we make this a bit more of a feature table?

| Retriever | Supports async | Supports upserts | Local | Package |

@eyurtsev dunno if "supports upserts" is something we want to document till its standardized